### PR TITLE
replace testidx with proper ioutil.TempDir

### PR DESCRIPTION
**Pull request by mschoch**

### CODE REVIEW

**Comment by mschoch** [comment]
I have some further cleanup planned for some follow-on PRs.

File: test/versus_score_test.go [hunk 1]
```diff
@@ -48,9 +49,19 @@ func TestDisjunctionSearchScoreIndexWithCompositeFields(t *testing.T) {
 
 func disjunctionQueryiOnIndexWithCompositeFields(indexName string,
 	t *testing.T) []*search.DocumentMatch {
+	tmpIndexPath, err := ioutil.TempDir("", "bleve-testidx")
```
Comment (abhinavdangeti) [inline test/versus_score_test.go:52]: Why not use the same createTmpIndexPath(..)-defer cleanupTmpIndexPath(..) code path here?

File: index_test.go [hunk 2]
```diff
@@ -42,15 +43,30 @@ import (
 	"github.com/blevesearch/bleve/index/upsidedown"
 )
 
+type Fatalfable interface {
```
Comment (abhinavdangeti) [inline index_test.go:46]: I read Fatalf-able as Fatal-Fable :)

File: test/versus_score_test.go (hunk 1 above)
Comment (mschoch) [reply test/versus_score_test.go:52]: It's in a different package, and this is the only usage in this package, so I figured more clear to do it directly.  If you think copy/pasting those funcs is better, I don't feel too strongly about it.

File: index_test.go (hunk 2 above)
Comment (mschoch) [reply index_test.go:46]: Yeah at first I didn't like it, but it grew on me.

File: test/versus_score_test.go (hunk 1 above)
Comment (abhinavdangeti) [reply test/versus_score_test.go:52]: Fair enough :)

**Comment by abhinavdangeti** [comment]
I'm good with this PR!

**Pull request merged**